packit_service/worker/build/koji_build.py

```diff
@@ -136,12 +148,21 @@ def run_build(
             logger.debug("No targets set for koji build, using rawhide.")
             target = "rawhide"
 
-        out = self.api.up.koji_build(
-            scratch=self.is_scratch,
-            nowait=True,
-            koji_target=target,
-            srpm_path=self.srpm_path,
-        )
+        try:
+            out = self.api.up.koji_build(
+                scratch=self.is_scratch,
+                nowait=True,
+                koji_target=target,
+                srpm_path=self.srpm_path,
+            )
+        except PackitCommandFailedError as ex:
+            logger.warning(
```

Comment:
Wouldn't make more sense to log an error if we raise an error?

Reply:
This code is located in packit -- we already do it but without the stout/stderr.

(I'll take a look at it.)

Reply:
OK, I understood your comment as why we don't do this in a place the original error is raised.